src/ppom-storage.test.ts

```diff
@@ -0,0 +1,287 @@
+import { StorageBackend, PPOMStorage, StorageKey } from './ppom-storage';
+
+Object.defineProperty(globalThis, 'crypto', {
+  value: {
+    subtle: {
+      digest: () => new ArrayBuffer(12),
+    },
+  },
+});
+
+const checksum =
+  '409a7f83ac6b31dc8c77e3ec18038f209bd2f545e0f4177c2e2381aa4e067b49';
+class MockStorageBackend implements StorageBackend {
+  data: undefined;
```

Comment:
This assigns the `undefined` type to `data`, which is probably not what you want.

Reply:
I changed this code and added a builder function for this.